src/FixedDataTableCell.js

```diff
@@ -233,62 +205,46 @@ class FixedDataTableCell extends React.Component {
         'public/fixedDataTableCell/alignRight': props.align === 'right',
         'public/fixedDataTableCell/highlighted': props.highlighted,
         'public/fixedDataTableCell/main': true,
-        'public/fixedDataTableCell/hasReorderHandle': !!props.onColumnReorder,
-        'public/fixedDataTableCell/reordering': this.state.isReorderingThisColumn,
       }),
       props.className,
     );
 
-    var columnResizerComponent;
-    if (props.onColumnResize) {
-      var columnResizerStyle = {
-        height
-      };
-      columnResizerComponent = (
-        <div
-          className={cx('fixedDataTableCellLayout/columnResizerContainer')}
-          style={columnResizerStyle}
-          onMouseDown={this._onColumnResizerMouseDown}
-          onTouchStart={this.props.touchEnabled ? this._onColumnResizerMouseDown : null}
-          onTouchEnd={this.props.touchEnabled ? this._suppressEvent : null}
-          onTouchMove={this.props.touchEnabled ? this._suppressEvent : null}>
-          <div
-            className={joinClasses(
-              cx('fixedDataTableCellLayout/columnResizerKnob'),
-              cx('public/fixedDataTableCell/columnResizerKnob'),
-            )}
-            style={columnResizerStyle}
-          />
-        </div>
-      );
-    }
-
-    var columnReorderComponent;
-    if (props.onColumnReorder) { //header row
-      columnReorderComponent = (
-        <FixedDataTableColumnReorderHandle
-          columnKey={this.columnKey}
-          touchEnabled={this.props.touchEnabled}
-          onMouseDown={this._onColumnReorderMouseDown}
-          onTouchStart={this._onColumnReorderMouseDown}
-          height={height}
-          {...this.props}
-        />
-      );
-    }
-
     var cellProps = {
       columnKey,
       height,
-      width
+      width,
     };
+    if (this.props.isHeader) {
```

Comment:
This is a lot to pass to every header.  I wonder if we could clean up this API some.  That said, I don't have any suggestions, so if no improvements jump out, no worries.